### PR TITLE
LL-6657 Rebrand - Messages > Alerts (with stories)

### PR DESCRIPTION
**Pull request by nparigi-ledger**
## 🦒 Context (issues, jira)

https://ledgerhq.atlassian.net/browse/LL-6657

## 💻  Description / Demo (image or video)

Simple "Alert" component

https://user-images.githubusercontent.com/89014981/132230917-2938bece-1f3e-41bc-8d43-39eaa723cfce.mp4

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [X] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [x] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [X] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

File: ui-lib/src/components/Message/Alert/index.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  margin-right: 12px;
+  display: flex;
+  align-items: center;
+`;
+
+const getIcon = (type: AlertType) => {
+  switch (type) {
+    case "info":
+      return (
+        <StyledIconContainer>
```
Comment (LFBarreto) [inline ui-lib/src/components/Message/Alert/index.tsx:26]: you're repeating the container quite a bit why not put it in the template like 
```
<StyledIconContainer>
 { getIcon(type) }
</StyledIconContainer>
```

**Review by LFBarreto — CHANGES_REQUESTED** [review]

File: ui-lib/src/components/Message/Alert/index.tsx (hunk 1 above)
Comment (LFBarreto) [reply ui-lib/src/components/Message/Alert/index.tsx:26]: also i tend to prefer the indexer notation than a switch like `icons[type]` to access a specific value :grimacing:

Comment (nparigi-ledger) [reply ui-lib/src/components/Message/Alert/index.tsx]: I just made the recommanded change, it's simpler and clearer :ok_hand:

**Comment by nparigi-ledger** [comment]
Updated to new palette

**Review by valpinkman — APPROVED** [review]

**Pull request merged**